ajax/libs/kbw.svg/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "kbw.svg",
+  "filename": "jquery.svg.min.js",
+  "description": "This plugin allows you to control/interact with SVG elements within your HTML documents. * Native SVG support in major browsers (but only IE9+). * Create a new SVG element or wrap an existing inline SVG element. * Animate SVG attributes with the svganim module. * Apply jQuery class/attribute/event functionality to SVG elements with the svgdom module. * Draw graphs with the svggraph module. * Plot functions with the svgplot module.",
```

Comment:
I think it's better to not use "list of description". How about `A jQuery plugin that lets you interact with an SVG canvas.` from the offical website?
http://keith-wood.name/svg.html